webapp/scripts/modules/time_map.js

```diff
@@ -47,9 +47,14 @@ module.exports = class {
     if (!fs.existsSync(`${GRASS}/variables/roads_speed`)) {
       fs.copyFileSync(`${GRASS}/variables/defaults/roads_speed_defaults`, `${GRASS}/variables/roads_speed`)
     }
-    this.roadsSpeed = fs.readFileSync(`${GRASS}/variables/roads_speed`).toString().trim().split('\n')
+    if (!fs.existsSync(`${GRASS}/variables/roads_speed_wal`)) {
+      fs.copyFileSync(`${GRASS}/variables/defaults/roads_speed_wal_defaults`, `${GRASS}/variables/roads_speed_wal`)
```

Comment:
There is no need to abbreviate things. Write "walking" and "bicycle" instead of "wal" and "bic" for clarity.